### ISSUE TITLE
Couldn't find spec/features in Guardfile

### ISSUE THREAD

**Issue by ghost** [alexpeachey/rails_spec_harness#1]
This is probably not an issue, I was just wondering what your decision was to not include `spec/features` in the default `Guardfile`. I know they're super slow compared to the other tests.

What do you think of a `Guardfile` like this?

``` ruby
guard :rspec, notification: true, cmd: 'rspec --color --format nested --tty', all_after_pass: true, spec_paths: ["spec/actions", "spec/decorators", "spec/forms", "spec/models", "spec/services"] do
  watch(%r{^spec/(actions|decorators|forms|models|services)/.+_spec\.rb$})
  watch(%r{^spec/(actions|decorators|forms|models|services)_spec_helper\.rb$}) { |m| "spec/#{m[1]}" }
  watch(%r{^spec/support_(actions|decorators|forms|models|services)/(.+)\.rb$}) { |m| "spec/#{m[1]}" }
  watch(%r{^app/(actions|decorators|forms|models|services)/(.+)\.rb$}) { |m| "spec/#{m[1]}/#{m[2]}_spec.rb" }

  watch('/spec/base_spec_helper.rb') { ["spec/actions", "spec/decorators", "spec/forms", "spec/models", "spec/services"] }
  watch(%r{^spec/support_common/(.+)\.rb$}) { ["spec/actions", "spec/decorators", "spec/forms", "spec/models", "spec/services"] }
end

guard :rspec, cmd: 'rspec --color --format nested --tty', spec_path: 'spec/features' do
  watch(%r{^spec/features/.+_spec\.rb$})
end
```

On a tangent, isn't it duplicative to have both the rspec flags in the cmd hash and in the `.rspec` file?

Thanks! This gem is awesome.

**Comment by alexpeachey** [comment]
We decided to not include the spec/features in guard primarily for the reason you stated, they are slow and we don't want them triggered all the time while working a level deeper.

Since our normal flow is to write a feature spec or two until we are "stuck" and have to go deeper, then after diving down, resurface and run the feature spec, running `rspec spec/features` by hand isn't much of a hinderance.

That being said, I don't have a problem with your compromise and only spinning through features when a feature spec file is changed gets around that issue of them re-firing all the time when we don't want them to.

And to your tangent, yes, that is duplicative and we could probably skip the flags in guard. A lot of this "harness" was pulled together from various repetitive patterns on several projects so there are still cases where we have unneeded files, lines of config, or other options that need to get cleaned up.

We've also been noticing new things emerging where we've been using the harness and find ourselves modifying configs in certain ways. I've got a short list waiting to be back-ported into the gem.
